v3/src/components/component-title-bar.tsx

```diff
@@ -43,10 +43,11 @@ export const ComponentTitleBar = observer(function ComponentTitleBar(
       </Editable>
       <Flex className="header-right">
         <Button className="component-minimize-button">
-          <MinimizeIcon className="component-minimize-icon" title={t("DG.Component.minimizeComponent.toolTip")}/>
+          <MinimizeIcon className="component-minimize-icon" title={t("DG.Component.minimizeComponent.toolTip")}
+          data-testid="component-minimize-icon"/>
```

Comment:
Is it important for the `data-testid` to be on the icon? It would be more consistent with the close button if it were on the `Button`.

Reply:
@kswenson , the reason I added the `data-testid` on the minimize icon is because unlike the close button where the `title` is on the button, for the minimize button, the `title` is on the icon and I'm checking these titles for the tooltips. Please let me know if it makes sense to move the `title` to the button for minimize and I can change both the `title` and `data-testid` on it.

Reply:
I would think that the `title` _and_ the `data-testid` should be on the button -- it's the button that has the behavior that the tooltip is describing, not the icon.